code/src/components/MessageList.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react'
+
+import moment from 'moment'
+
+export const MessageList = () => {
+    const MESSAGES_URL = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'
+    const [messages, setMessages] = useState([]);
+
+
+    useEffect(() => {
+        fetchMessages()
+    }, [])
+
+    const fetchMessages = () => {
+        fetch(MESSAGES_URL)
+            .then((res) => {
+                return res.json();
+            })
+            .then((data) => {
+                const filtredMessages = data.filter((message) => message.message)
+
+                setMessages(filtredMessages)
+            })
+    }
+
+    const likedHeart = (messageId) => {
+        fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${messageId}/like`, {
+            method: "POST",
+            headers: { "Content-Type": "application/json" }
+        }).then(() => {
+            fetchMessages()
+        })
+    }
+
+
+    return (
+        <>
+            <div className="message-list">
+                {messages.map((message) => {
+                    return (
+                        <div key={message._id} className="message-section">
+                            <p className="message" key={message._id}>
+                                {message.message}
+                                <br></br>
```

Comment:
We shouldn't use `<br />` tags, they are deprecated. You should use proper styles in CSS file in order to achieve same result. Maybe `display` property will help?